roles/ncn-common-setup/defaults/main.yml

```diff
@@ -28,4 +28,8 @@ services_metal:
   - name: metal-iptables
     enabled: no
     state: stopped
-    
+cray_dns_servers:
+  - 172.31.84.40
+  - 172.30.84.40
+required_suse_extensions:
```

Comment:
nit: a comment for where these are located would be helpful, I think these print out with `SUSEConnect --list-extensions`.

Maybe a link to this page?
```yaml
# For help see this page for listing extensions:
# https://www.suse.com/support/kb/doc/?id=000018564
required_suse_extensions:
```

For the other reviewers, this is how you could do it quickly:
```bash
docker run -it artifactory.algol60.net/csm-docker/stable/csm-docker-sle:15.3
SUSEConnect -r <reg code>
SUSEConnect --list-extensions
```